src/jit/gentree.h

```diff
@@ -1821,7 +1817,10 @@ struct GenTree
 
     // Requires "childNum < NumChildren()".  Returns the "n"th child of "this."
     GenTreePtr GetChild(unsigned childNum);
-
+#if 0
+    // Returns the i'th child of a dynamic block node, in execution order.
+    GenTreePtr* GetDynBlkOperandInOrder(int index);
+#endif
     // Returns an iterator that will produce the use edge to each operand of this node. Differs
```

Comment:
Oops; thought I'd cleaned those up.

---
In reply to: [77247862](https://github.com/dotnet/coreclr/pull/7027#discussion_r77247862) [](ancestors = 77247862)